docs/project-setup/filesystem-sync.md

```diff
@@ -0,0 +1,73 @@
+# Filesystem Sync with Unison
+
+The default NFS filesystems that Outrigger provides are very easy to setup and easy to share code and other files into
```

Comment:
The default NFS filesystems that Outrigger sets up provide easy sharing of code and files with your project containers. However, NFS can be slow when writing, reading, or scanning thousands of files when compared to native filesystem performance. Some modern toolkits and package managers favor large numbers of small files and libraries. Maintaining high performance of a build becomes challenging when your code base is on a volume mounted NFS share.